src/utils/changelog.ts

```diff
@@ -175,3 +181,193 @@ export function prependChangeset(
 
   return markdown.slice(0, startIdx) + newSection + markdown.slice(startIdx);
 }
+
+interface Commit {
+  hash: string;
+  title: string;
+  hasPRinTitle: boolean;
+  pr: string | null;
+  milestone: string | null;
+}
+
+interface Milestone {
+  title?: string;
+  description?: string;
+  state?: 'OPEN' | 'CLOSED';
```

Comment:
How about doing `enum State` instead? We use it in few places so would be easier to refactor in the future if necessary.

Reply:
Since this code is fairly isolated here, I'd prefer deferring the enum to a later stage. If you feel strongly, I can do it of course. Just don't see the immediate or even mid-term benefits.